bftengine/src/bftengine/SigManager.hpp

```diff
@@ -13,7 +13,7 @@
 #include "PrimitiveTypes.hpp"
 #include "assertUtils.hpp"
 #include "Metrics.hpp"
-#include "crypto_utils.hpp"
```

Comment:
do we need to include cryptopp_utils.hpp here?

Reply:
`cryptopp_utils.hpp` removed.